fetools-app/src/pages/ColorGradient.jsx

```diff
@@ -212,9 +241,14 @@ export default function ColorGradient() {
     </>
   );
 
-  function handleQuery(gradientColors) {
-    color = color.slice(1);
-    setSearchParams({ color });
+  function handleQuery(gradientColorsArr) {
+    const colors = {}
+
+    gradientColorsArr.forEach(({colorStr},idx)=>{
+      colors[`color${idx}`] = getColorString(colorStr, 'hex').replace('#', '')
+    })
+
+    setSearchParams(colors);
```

Comment:
The search params need to also include position for each color as well as rotation.